tests/testdata/buf.build/community/infobloxopen-gorm/v1.1.2/eliza/plugin.sum

```diff
@@ -0,0 +1 @@
+h1:47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=
```

Comment:
I haven't used this plugin before and our test inputs don't have the required options (`gorm.*`) specified, so currently there is no code generated with our tests. We might need to manually verify the plugin against a more complete example to ensure it will do the right thing when added to the BSR.

Reply:
Is there something I can provide?

Reply:
You'd need to modify the plugin so that the base case works when no options are specified - this should not result in an error (perhaps it results in no code output)